client/src/TagExplorer/resource_scheme.js

```diff
@@ -273,14 +276,16 @@ export class ResourceScheme extends AbstractExplorerScheme {
     } else if (type === "set_hierarchy_scheme") {
       return { ...state, hierarchy_scheme: payload };
     } else if (type === "column_header_click") {
-      const { is_descending, sort_col } = state;
-      const clicked_col = payload;
-
+      const { sort_col } = state;
+      const clicked_col = payload.col_key;
+      const is_descending_bool = payload.is_descending_bool;
       const mods =
         clicked_col === sort_col
-          ? { is_descending: !is_descending }
-          : { is_descending: true, sort_col: clicked_col };
-
+          ? { is_descending: is_descending_bool === "DESC" }
```

Comment:
Oh, is this some old code or is `is_descending_bool` sometimes string valued after all?